### PR TITLE
Model Inference Workflows

### PR DESCRIPTION
**Pull request by BryanRumsey**
### Changes
- [x] Add legend for v-lines
- [x] Change current values line color to {color: 'black', alpha: 0.75}
- [x] Change inferred values line color to {color: 'black', alpha: 0.75} (dashed line)
- [x] Change Y-Title to "Accepted Sample" (histogram) "Probability" (pdf)
- [x] Change X-Title to "Parameter Values"
- [x] Add instructive text to plot round
- [x] Plot pdf curves with a alternate scale.
- [x] Add legend to intersection plots

### Bugs
- [x] Download as .png button fails.
- [x] Download as .json is downloading extra content.

### CODE REVIEW

File: stochss/handlers/util/sciope_notebook.py [hunk 1]
```diff
@@ -119,72 +130,163 @@ def __create_me_header_cells(self, cells):
...
+                f"{pad*2}for label, formula in definitions.items():",
+                f"{pad*3}evaluations[label] = eval(formula, dict(), result.data)",
+                f"{pad*2}trajectories.append(gillespy2.Trajectory(",
+                f"{pad*3}data=evaluations, model=result.model, solver_name=result.solver_name, rc=result.rc",
+                f"{pad*2}))", f"{pad}processed_results = gillespy2.Results([evaluations])",
+                f"{pad}return processed_results.to_array().swapaxes(1, 2)[:,1:, :]"
+            ]
+            lines = []
+            for feature_calculator in self.settings['inferenceSettings']['summaryStats']:
+                line = f"{pad*2}{feature_calculator['name']}='{feature_calculator['formula']}',"
```
Comment (BryanRumsey) [inline stochss/handlers/util/sciope_notebook.py:259]: Missing `lines.append(line)`

File: stochss/handlers/util/sciope_notebook.py [hunk 2]
```diff
@@ -119,72 +130,163 @@ def __create_me_header_cells(self, cells):
...
-        ))
+        if self.settings['inferenceSettings']['summaryStatsType'] == "identity":
+            ident_lines = [
+                f"{pad}definitions = dict(", f"{pad*2}time='time'", f"{pad})", "", f"{pad}trajectories = []",
+                f"{pad}for result in raw_results:", f"{pad*2}evaluations = dict()",
+                f"{pad*2}for label, formula in definitions.items():",
+                f"{pad*3}evaluations[label] = eval(formula, dict(), result.data)",
+                f"{pad*2}trajectories.append(gillespy2.Trajectory(",
+                f"{pad*3}data=evaluations, model=result.model, solver_name=result.solver_name, rc=result.rc",
+                f"{pad*2}))", f"{pad}processed_results = gillespy2.Results([evaluations])",
```
Comment (BryanRumsey) [inline stochss/handlers/util/sciope_notebook.py:254]: `gillespy2.Results` should be passed `trajectories` not `[evaluations]`

**Comment by BryanRumsey** [comment]
## Changes
- change extension '.obsd' to '.odf'.
- change 'load_obs_data' to 'load_observed_data'.
- Add text to observed data section to inform users of accepted file types.

**Review by seanebum — APPROVED** [review]

File: client/settings-view/templates/inferenceSettingsView.pug [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+                div(data-hook="import-chevron")
+
+                  svg(xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" viewBox="0 0 512 512")
+
+                    path(d="M233.4 406.6c12.5 12.5 32.8 12.5 45.3 0l192-192c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L256 338.7 86.6 169.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3l192 192z")
+
+            div.card-body
+
+              p.mb-0
+                | Valid files are <b>Observed Data Files</b> (a directory with the postfix .odf) compressed to a zip archive (.zip) or <b>Observed Data File</b> (.csv)
```
Comment (seanebum) [inline client/settings-view/templates/inferenceSettingsView.pug:88]: I do think this is a little weird... the Bold followed by the parenthesis implies a relationship between the bolded words and the extension.... but .csv is not technically connected to "Observed Data File"

**Review by briandrawert — APPROVED** [review]

**Pull request merged**